content/en/docs/quarks-secret/_index.md

```diff
@@ -113,9 +120,6 @@ And each destination which is indicated needs to have a `Quarks Secret` of `copy
 
 The examples copies the generated `gen-secret` secret content into `copied-secret`  inside the `COPYNAMESPACE` namespace.
 
-## See also
```

Comment:
Same here.. not sure about this, but feel free to drop. On the other hand a section with just one phrase won't look very neat :sunglasses: